### PR TITLE
Added space for Shard

### PR DESCRIPTION
**Pull request by InfinityWallet**
#### Hi! What is your PR about?

- [x] Add images for your space
- [x] Add custom domain for your space
- [ ] Add alias for migrate your space
- [x] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/shardcoin.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.shardcoin {
+	--primary-color: #019afb;
+    --bg-color: #ffffff;
+    --text-color: #606988;
+    --link-color: #019afb;
+    --heading-color: #019afb;
+    --border-color: #e1e3e7;
+    --header-bg: #f7f9fb;
```
Comment (gokaiorg) [inline skins/shardcoin.scss:8]: Could you indent your file? Thank you.
```suggestion
	--primary-color: #019afb;
  --bg-color: #ffffff;
  --text-color: #606988;
  --link-color: #019afb;
  --heading-color: #019afb;
  --border-color: #e1e3e7;
  --header-bg: #f7f9fb;
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: skins/shardcoin.scss (hunk 1 above)
Comment (InfinityWallet) [reply skins/shardcoin.scss]: Committed the changed

**Comment by gokaiorg** [comment]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?

**Comment by gokaiorg** [comment]
To update your fork with the remote version of snapshot-spaces you can try these GIT commands then commit your changes again.
`git remote add upstream https://github.com/snapshot-labs/snapshot-spaces.git`
`git fetch upstream`
`git checkout master`
`git merge upstream/master`

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**